fastapi_restful/cbv.py

```diff
@@ -111,6 +111,13 @@ def _register_endpoints(router: APIRouter, cls: Type[Any], *urls: str) -> None:
     prefix_length = len(router.prefix)  # Until 'black' would fix an issue which causes PEP8: E203
     for route in cbv_routes:
         router.routes.remove(route)
+        # remove any tags that were previously assigned by router to route
+        if isinstance(route, APIRoute):
+            for tag in router.tags:
+                try:
+                    route.tags.remove(tag)
+                except ValueError:
+                    pass

```

Comment:
Maybe `route.tags = list(set(router.tags + route.tags))` or something similar?

Reply:
I've changed it to `route.tags = list(set(route.tags) - set(router.tags))` as we need to remove the tags from the router, not merge them.  The duplication happens later when the route is re-added to the router.